testsuite/integration/microprofile-tck/jwt/pom.xml

```diff
@@ -213,6 +213,7 @@
                     <systemPropertyVariables>
                         <microprofile.jvm.args>${microprofile.jvm.args}</microprofile.jvm.args>
                     </systemPropertyVariables>
+                    <argLine>-Djdk.security.defaultKeySize=AES:128</argLine>
```

Comment:
Should this be done in a profile?

Reply:
You mean something like JDK19 specific profile @bstansberry  ?
IMO it is not necessary. The default AES key size changed from 128 to 256
in JDK19 and this is just a workaround for testsuite until it will be properly fixed in JWT TCK.